Makefile

```diff
@@ -53,7 +53,8 @@ install: clean
     	brew install openssl; \
         env LDFLAGS="-L$(shell brew --prefix openssl)/lib" CFLAGS="-I$(shell brew --prefix openssl)/include" pipenv install --python 3.6 --dev; \
         else \
-        	pipenv install --python 3.6 --dev; \
+		pipenv lock --python 3.6 --hashes; \
```

Comment:
Since this recipe isn't used in the Dockerfile, and is, therefore, a dev environment recipe, locking should happen just before commit time, not before install. Also, the `--hashes` flag is not part of pipenv.

Reply:
The hashes flag is part of pipenv when using the lock command.

Anyway, perhaps this should be a make target? Thoughts?

Reply:
[`--hashes` flag removed](https://github.com/kennethreitz/pipenv/blob/8deba2e33e945f162b2396bef20f5b31612a032f/HISTORY.txt#L386)